microsoft-365/compliance/import-non-office-365-data-into-advanced-ediscovery.md

```diff
@@ -50,36 +50,34 @@ Using the upload Non-Office 365 feature as described in this procedure requires
 1. As an eDiscovery Manager or eDiscovery Administrator, open **eDiscovery**, and open the case that the non-Office 365 data will be uploaded to. If you need to create a case, see [Manage eDiscovery cases in the Security &amp; Compliance Center](ediscovery-cases.md)
     
 2. Click **Switch to Advanced eDiscovery**
-    
-3. Under **Source type** select **Non-Office 365 data**.
-    
-4. Click **Add container**. Name the container and add a description.
-    
-5. Select the newly added container from the container list and copy the URL that appears in the container details pane and then close the pane
-    
-6. Open a command prompt as an administrator and change directory to folder where you have AzCopy installed..
-    
-7. Construct the AzCopy command line to upload the files like this:
-    
-    AzCopy /Source:" *full path to root folder on local machine*  " /Dest:"  *container URL up to but not including the ?*  " /DestSAS:"  *remainder of the container url from the ? to the end*  " /S. 
-    
-    For example, using these values: 
-    
-  - root folder - C:\Collected Data 
-    
-  - container url - https://zoomsabcprodeuss114.blob.core.windows.net/ingestion53d059efe5f74784afb308f66cdebf17?sv=2015-04-05&amp;sr=c&amp;si=NonOfficeData15%7C0&amp;sig=Bk5INP8CUfv1y4CSJiJl3pJt3Ekvu8GS3P8NkOvoQxA%3D
-    
-    the AzCopy command line syntax would be:
-    
-     `AzCopy /Source:"C:\CollectedData" /Dest:"https://zoomsabcprodeuss114.blob.core.windows.net/ingestion53d059efe5f74784afb308f66cdebf17" /DestSAS:"?sv=2015-04-05&amp;sr=c&amp;si=NonOfficeData15%7C0&amp;sig=Bk5INP8CUfv1y4CSJiJl3pJt3Ekvu8GS3P8NkOvoQxA%3D" /S`
-    
+
+3. Select **Review Sets** from the menu
+
+4. Select and existing Review Set or click on **Add Review Set**
+
+5. Select **Manage review set**
+
+6. In the Non-Office 365 data card, click on **View Uploads**
+
+7. Click on **Upload files** to start the file upload wizard
+
+8. The first tab is the **1. Prepare step**. Click on **Next: Upload files**
+
+9. On the **2. Upload files** tab you will be promoted to download AzCopy.exe if you have not and then provide the path to the file location.  For example, `C:\Upload`.  This will give you the command to execute for AzCopy.exe, for example using `C:\Upload` you would see:
```

Comment:
```suggestion
9. On the **2. Upload files** tab you will be prompted to download AzCopy.exe if you have not done so already, and then to provide the path to the file location.  For example, `C:\Upload`.  This will give you the command to execute AzCopy.exe. For example, using `C:\Upload` you would see:
```